python/tvm/micro/session.py

```diff
@@ -86,7 +86,7 @@ def __init__(
 
         self._rpc = None
         self._graph_executor = None
-        self._enable_rpc_logger = False
+        self._enable_rpc_logger = True
```

Comment:
note to set back to `False` prior to completing PR

Reply:
Done!